elastic/kibana
# [AO] Update default action and recovery messages for observability rule types

Closes #157284, closes #158437

## Summary

This PR updates the default action and recovery message for observability rule types. Only uptime is excluded in this PR, I will add that after synching with the synthetics team.

[Here](https://docs.google.com/spreadsheets/d/15cJanv0xi4EKo1QZVXsi_NcFcPa_VQze3nQc4P8GZrQ/edit?pli=1#gid=0) you can find the list of default messages and the new proposals.
![image](https://github.com/elastic/kibana/assets/12370520/4f36e971-d8c2-47a9-ac25-9cae567db774)

## 🧪 How to test
- Ensure that `server.publicBaseUrl` is configured in kibana.dev.yml
- Create an observability rule with an action such as Email to use the default messages (both for alert and recover states)
- After an alert is triggered, check the generated action message, it should 
   - match the message mentioned in the list
   - contain an alert details link pointing to the Alerts page filtered for that specific alert
